share/fstab

```diff
@@ -1,19 +1,9 @@
 # /etc/fstab: static file system information.
-#
-# noatime turns off atimes for increased performance (atimes normally aren't 
-# needed); notail increases performance of ReiserFS (at the expense of storage 
-# efficiency).  It's safe to drop the noatime options if you want and to 
-# switch between notail / tail freely.
-#
-# The root filesystem should have a pass number of either 0 or 1.
-# All other filesystems should have a pass number of 0 or greater than 1.
-#
 # See the manpage fstab(5) for more information.
 #
-
-# <fs>			<mountpoint>	<type>		<opts>		<dump/pass>
-
-# NOTE: If your BOOT partition is ReiserFS, add the notail option to opts.
+#
```

Comment:
Looks like you're adding an extra empty line here.

Reply:
Right

Reply:
This still looks wrong?

```diff
@@ -13,7 +7,6 @@
 
 # <fs>			<mountpoint>	<type>		<opts>		<dump/pass>
 
-# NOTE: If your BOOT partition is ReiserFS, add the notail option to opts.
 #
 # NOTE: Even though we list ext4 as the type here, it will work with ext2/ext3
 #       filesystems.  This just tells the kernel to use the ext4 driver.
```